<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimNodeClass</class>
 <widget class="QWidget" name="SimNodeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimNode</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>节点</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <item>
       <widget class="QComboBox" name="comboBox_subscriber">
        <item>
         <property name="text">
          <string>01</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>02</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>03</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>04</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>05</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>06</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>07</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>08</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>09</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QPushButton" name="pushButton_subscriber">
          <property name="text">
           <string>开始</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_only_subscriber">
          <property name="text">
           <string>只接收</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>查询反馈</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QPushButton" name="pushButton_feed_nodesearch">
             <property name="text">
              <string>反馈节点步长消息</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_feed_nodesearchstate">
             <property name="text">
              <string>反馈节点状态消息</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QRadioButton" name="radioButton_sub_init">
                 <property name="text">
                  <string>初始态</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_sub_run">
                 <property name="text">
                  <string>运行态</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <widget class="QRadioButton" name="radioButton_sub_pause">
                 <property name="text">
                  <string>暂停态</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_sub_error">
                 <property name="text">
                  <string>故障态</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>步长</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_subscriber_step"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>毫秒</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QGroupBox" name="groupBox_7">
            <property name="title">
             <string>协同控制反馈</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QPushButton" name="pushButton_feed_run">
               <property name="text">
                <string>协同推进回令</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_feedbackalway">
               <property name="text">
                <string>自动</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_feed_pause">
               <property name="text">
                <string>协同暂停回令</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_feed_stop">
               <property name="text">
                <string>协同终止回令</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_8">
            <property name="title">
             <string>跳转反馈</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QPushButton" name="pushButton_feed_snapshoot">
               <property name="text">
                <string>反馈节点快照信息</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QtMyTextEdit" name="textEdit_subscriber"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtMyTextEdit</class>
   <extends>QTextEdit</extends>
   <header>QtMyTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="SimNode.qrc"/>
 </resources>
 <connections/>
</ui>
